refresh.template.py

```diff
@@ -93,6 +93,23 @@ def _print_header_finding_warning_once():
     Continuing gracefully...""")
 _print_header_finding_warning_once.has_logged = False
 
+def _file_exists(source_file, warn_missing_generated = True):
```

Comment:
Thanks for your care in preserving this for swift :) It really does help a lot of people with generated code.

[Small tweak: If we're always leaving warn_missing_generated = True, what about just omitting the parameter, switching the return values, and naming it something like `_warn_if_file_doesnt_exist`]

Reply:
Done

https://github.com/hedronvision/bazel-compile-commands-extractor/commit/4fbd4a38987aae6801b637dac43a88729b951b51